lib/cog/command/ambiguous_alias_checker.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Cog.Command.AmbiguousAliasChecker do
+  alias Cog.Repo
+  alias Cog.Queries.Alias
+  alias Cog.Models.Command
+  alias Piper.Command.Ast
+
+  def ambiguous_command?(%Ast.Invocation{command: "site:" <> _}),
+    do: false
+  def ambiguous_command?(%Ast.Invocation{command: "user:" <> _}),
```

Comment:
I like that. Could that warrant another PR though? That seems like that belongs in alias management rather than execution.